coderd/audit_test.go

```diff
@@ -31,7 +31,8 @@ func TestAuditLogs(t *testing.T) {
 		user := coderdtest.CreateFirstUser(t, client)
 
 		err := client.CreateTestAuditLog(ctx, codersdk.CreateTestAuditLogRequest{
-			ResourceID: user.UserID,
+			OrganizationID: uuid.New(),
```

Comment:
Oh interesting.  I did not really think too hard on this since I was just moving out the `uuid.New()` from `CreateTestAuditLog()` out to the test, but you make a good point that most of these do not even need to set the ID  for the test to pass.  I will remove them!